### ISSUE TITLE
Library shuffle when closing checkbox clears after closing sideboard.

### ISSUE THREAD

**Issue by Psithief** [Cockatrice/Cockatrice#591]
After #549, the library's `shuffle when closing` checkbox clears itself.

## To reproduce:
1. Start a local 1 player game
2. Open Library. Ensure that `pile view` is unchecked and `shuffle when closing` is checked.
3. Close Library (shuffle message appears in log)
4. Open Sideboard.
5. Close sideboard.
6. Open Library again. Shuffle checkbox is now unchecked.
7. Close Library (shuffle message absent from log)

```
Player 1 is looking at his library.
Player 1 stops looking at his library.
Player 1 shuffles his library.
Player 1 is looking at his sideboard.
Player 1 stops looking at his sideboard.
Player 1 is looking at his library.
Player 1 stops looking at his library.
```

build b06b818

**Comment by Psithief** [comment]
Also check graveyard and exile for the same behaviour.

**Comment by poixen** [comment]
This was introduced because of the storing of checkbox data. Looking into it now. This is high prio.

**Comment by poixen** [comment]
Fixed in #595

**Comment by Psithief** [comment]
@poixen 
I'm just curious to know whether the shuffle checkbox is on and saved on by default?
I am not sure how to clear my preferences to check.
